packages/react/src/components/accordion/__snapshots__/Accordion.test.tsx.snap

```diff
@@ -46,7 +46,7 @@ exports[`<Accordion /> spec renders the component 1`] = `
     </div>
     <div
       aria-labelledby="accordion-1-heading"
-      class="accordionContent contentWithCloseButton"
+      class="accordionContent"
```

Comment:
In `Accordion.tsx` and its css, the`contentWithCloseButton` was removed, `contentWithoutCloseButton` added and conditional `closeButton && styles.contentWithCloseButton` changed to `!closeButton && styles.contentWithoutCloseButton´. But the `contentWithCloseButton` does not appear in the snapshot.

So it is not tested. I'd add test for it.

Reply:
Yeah, will do. Good spot.